### ISSUE TITLE
Detoured CreateFileW returning same handle for different files.

### ISSUE THREAD

**Issue by miyu** [the-dargon-project/the-dargon-project#14]
trinket-dim's CreateFileW hook (or something around it) seems to cause m_trampCreateFileW to return the same handle when CreateFileWing (which can open files as well) different paths. This leads to, for example, what seems like memory stomping, because when opening game.cfg, the handle might already exist and be pointing to a raf.dat file.

This is a pretty strange problem because the duplicate handle doesn't actually point to whatever we passed to m_trampCreateFileW; instead, it points to whatever was already opened. Further work needs to look into whether this is actually increasing Windows' reference count for that handle.

**Comment by miyu** [comment]
The fix is pretty jank: https://github.com/the-dargon-project/the-dargon-project/blob/df3674134c9761df1ee96d05cf8bdfe12ae8a22a/DargonInjectedModule/Subsystems/FileSubsystem.cpp#L364

Tl;dr need to figure out a way to tell apart LoL's IO from whatever else (Windows?) is doing. We're still getting duplicate file handles; we're just tossing them all away with this simple filter.
